src/js/index.js

```diff
@@ -1233,3 +1233,253 @@ export const hpe = deepFreeze({
 });
 
 export const { colors } = hpe.global;
+
+export const DarkCodeTheme = {
```

Comment:
Just a general naming thing, should we do camel case? `darkCodeTheme` and `lightCodeTheme`?

Reply:
Yes I will update the names to be camel case